src/github.com/getlantern/flashlight/ui/ui.go

```diff
@@ -0,0 +1,72 @@
+package ui
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"path"
+	"path/filepath"
+
+	"github.com/getlantern/golog"
+	"github.com/getlantern/tarfs"
+	"github.com/skratchdot/open-golang/open"
+)
+
+const (
+	localResourcesPath = "../ui/app"
```

Comment:
@myleshorton Just changed this to make it relative to the GOPATH so it should work no matter where you run flashlight.  Can you please try again?